addons/website_sale/controllers/main.py

```diff
@@ -899,11 +903,16 @@ def payment_validate(self, transaction_id=None, sale_order_id=None, **post):
             composer_values.update(composer_obj.onchange_template_id(cr, SUPERUSER_ID, None, *template_values, context=context).get('value', {}))
             if not composer_values.get('email_from') and uid == request.website.user_id.id:
                 composer_values['email_from'] = request.website.user_id.company_id.email
-            for key in ['attachment_ids', 'partner_ids']:
+            for key in ['partner_ids']:
                 if composer_values.get(key):
                     composer_values[key] = [(6, 0, composer_values[key])]
             composer_id = composer_obj.create(cr, SUPERUSER_ID, composer_values, context=email_ctx)
+            attachments =  composer_obj.browse(cr, SUPERUSER_ID, [composer_id],context=context).attachment_ids
```

Comment:
Because from [onchange_template_id] (https://github.com/odoo-dev/odoo/blob/master-digital-product-dja/addons/website_sale/controllers/main.py#L903) method, attachment for quotation/sale order will always be created and will be sent along with the mail. And as now we are not sending default template of SO/Quotation, we don't want to attach related SO/Quotation in the mail.